snooty.toml

```
@@ -4,5 +4,14 @@ title = "Visual Studio Extension"
 [constants]
 version = 1.0
 full-version = "{+version+}.0"
+
 name = "MongoDB Visual Studio Extension"
-home-page = "https://github.com/BorisDog/mongo-csharp-analyzer"
\ No newline at end of file
+home-page = "https://github.com/BorisDog/mongo-csharp-analyzer"
+query-api = "MongoDB Query API"
+
+csharp-version = 2.14
+csharp-docs = "https://mongodb.github.io/mongo-csharp-driver/{+csharp-version+}/"
+
+driver-long = "MongoDB .NET driver"
+driver = ".NET driver"
```

comment: this seems like it might be better as "driver-short"
```suggestion
driver-short = ".NET driver"
```